main/FatesHistoryInterfaceMod.F90

```diff
@@ -1306,6 +1304,27 @@ subroutine update_history_dyn(this,nc,nsites,sites)
     real(r8) :: frac_canopy_in_bin  ! fraction of a leaf's canopy that is within a given height bin
     real(r8) :: binbottom,bintop    ! edges of height bins
 
+    ! The following are all carbon states, turnover and net allocation flux variables
+    ! the organs of relevance should be self explanatory
+    real(r8) :: sapw_c
+    real(r8) :: struct_c
+    real(r8) :: leaf_c
+    real(r8) :: fnrt_c
+    real(r8) :: store_c
+    real(r8) :: alive_c
+    real(r8) :: total_c
+    real(r8) :: sapw_c_turnover
+    real(r8) :: store_c_turnover
+    real(r8) :: leaf_c_turnover
+    real(r8) :: fnrt_c_turnover
+    real(r8) :: struct_c_turnover
+    real(r8) :: sapw_c_net_art
```

Comment:
I think I should know what 'art' means already but I don't...

Reply:
"allocation and reactive transport".   It tracks the net changes in each pool due to those processes.

So losses or gains in storage due to retranslocation will be tracked in that flux pool.